lib/projections/omerc.js

```diff
@@ -181,8 +181,8 @@ export function inverse(p) {
   var u, v, Qp, Sp, Tp, Vp, Up;
   var coords = {};
   
-  p.x = (p.x * this.to_meter - this.x0) * (1.0 / this.a);
-  p.y = (p.y * this.to_meter - this.y0) * (1.0 / this.a);
+  p.x = (p.x - this.x0) * (1.0 / this.a);
```

Comment:
are we going to need to add some tests for non meter omerc projections ?

Reply:
I would think so.

I'll get back to this PR sometime next week, got pulled into some other stuff.